fhir-tools/pom.xml

```diff
@@ -290,63 +290,63 @@
-                            <minmemory>128m</minmemory>
+                            <minmemory>512m</minmemory>
                             <maxmemory>1g</maxmemory>
-                            <archive>
-                                <manifestEntries>
-                                    <Automatic-Module-Name>com.ibm.fhir</Automatic-Module-Name>
-                                </manifestEntries>
-                            </archive>
-                            <destDir>target/apidocs/${project.artifactId}</destDir>
                             <additionalOptions>
+                                <!-- makes the frames visible, disabled and going away eventually -->
                                 <additionalOption>--frames</additionalOption>
+                                <!-- Fixes the undefined redirect in search -->
+                                <additionalOption>--no-module-directories</additionalOption>
                             </additionalOptions>
-                            <bottom><![CDATA[Copyright 2019, <a href="http://www.ibm.com">IBM Corporation<a><br></br>FHIR&reg; is the registered trademark of HL7 and is used with the permission of HL7.]]></bottom>
+                            <header><![CDATA[<a href="/FHIR" class="bx--header__name" target="_parent"><span>IBM</span>&nbsp;FHIR®&nbsp;Server</a>]]></header>
+                            <bottom><![CDATA[Copyright 2019. <a href="http://www.ibm.com">IBM Corporation<a><br></br>FHIR&reg; is the registered trademark of HL7 and is used with the permission of HL7.]]></bottom>
                             <author>false</author>
-                            <detectJavaApiLink>false</detectJavaApiLink>
+                            <stylesheetfile>${project.build.directory}/../../fhir-parent/src/javadocs/stylesheet.css</stylesheetfile>
+                            <docletArtifact>
+                                <groupId>nl.talsmasoftware</groupId>
+                                <artifactId>umldoclet</artifactId>
+                                <version>2.0.5</version>
+                            </docletArtifact>
+                            <doclet>nl.talsmasoftware.umldoclet.UMLDoclet</doclet>
+                            <additionalparam>
+                                -umlSkipStandardDoclet true
+                                -umlLogLevel DEBUG
+                                -umlImageFormat png
+                            </additionalparam>
+                            <!-- skips the three entries -->
+                            <nohelp>true</nohelp>
+                            <nodeprecated>true</nodeprecated>
+                            <nodeprecatedlist>true</nodeprecatedlist>
                         </configuration>
                         <executions>
                             <execution>
-                                <id>aggregate</id>
+                                <id>attach-javadocs</id>
```

Comment:
here its `attach-javadocs` but in the PR description the commands are like `mvn site javadoc:aggregate -f fhir-parent/pom.xml -DskipTests  -X` ...just making sure its right

Reply:
oh, i see the `<phase>site</phase>` now...I guess the `mvn -Pfhir-javadocs site -f fhir-examples/pom.xml` will invoke this execution now.

Reply:
correct. that way it's not part of the main build.